### PR TITLE
data:cybergeekgyan

### PR DESCRIPTION
**Pull request by cybergeekgyan**

### CODE REVIEW

File: data/cybergeekgyan.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Data Science & CyberSecurity Enthusiast",
+  "avatar": "https://github.com/cybergeekgyan.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/cybergeekgyan",
+      "icon": "FaTwitter"
+    }
```
Comment (Panquesito7) [inline data/cybergeekgyan.json:12]:
```suggestion
    },
```

File: data/cybergeekgyan.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/gyanprakash-/",
+      "icon": "FaLinkedIn"
+    }
+{
```
Comment (Panquesito7) [inline data/cybergeekgyan.json:18]:
```suggestion
    },
    {
```

File: data/cybergeekgyan.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/cybergeekgyan",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/gyanprakash-/",
+      "icon": "FaLinkedIn"
```
Comment (Panquesito7) [inline data/cybergeekgyan.json:16]:
```suggestion
      "icon": "FaLinkedin"
```

File: data/cybergeekgyan.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    }
+{
+      "name": "Follow me on GitHub",
+      "url": "https://www.github.com/cybergeekgyan",
+      "icon": "FaGitHub"
```
Comment (Panquesito7) [inline data/cybergeekgyan.json:21]:
```suggestion
      "icon": "FaGithub"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Wrong icon names and missing commas.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**